src/state/transactions/updater.tsx

```diff
@@ -78,19 +40,7 @@ export default function Updater() {
           },
         })
       )
-      const tx = transactions[chainId]?.[hash]
 
-      if (tx.info.type === TransactionType.SWAP && trade) {
-        sendAnalyticsEvent(
```

Comment:
why remove this? it does seem useful to know whether the browser was open right?

Reply:
i think we agreed to remove it: https://uniswapteam.slack.com/archives/C03JAKVATU6/p1663003407549829?thread_ts=1662855832.386199&cid=C03JAKVATU6

Reply:
keeping it creates confusion because this gives an incomplete picture

Reply:
added back as per slack discussion